app/Filament/Resources/EmulatorResource/RelationManagers/EmulatorReleasesRelationManager.php

```diff
@@ -30,46 +30,73 @@ public function form(Form $form): Form
     {
         return $form
             ->schema([
-                Forms\Components\TextInput::make('version'),
+                Forms\Components\TextInput::make('version')
+                    ->placeholder('2.4.0')
+                    ->helperText('eg: "2.4.0" or "2407-68"')
+                    ->required(),
 
                 Forms\Components\DatePicker::make('created_at')
                     ->label('Release Date')
+                    ->helperText('When was this version of the emulator released?')
                     ->native(false)
-                    ->date(),
+                    ->date()
+                    ->required(),
 
-                Forms\Components\Toggle::make('stable'),
+                Forms\Components\Select::make('stable')
+                    ->label('Release Type')
+                    ->options([
+                        '1' => 'Stable',
+                        '0' => 'Beta',
+                    ])
+                    ->default('1')
+                    ->required()
+                    ->helperText('Stable releases can be set as minimum and are reported via the Connect API. Beta releases are experimental.'),
 
-                Forms\Components\Toggle::make('minimum'),
+                Forms\Components\Toggle::make('minimum')
+                    ->label('Set as recommended minimum version')
+                    ->helperText('Sets the minimum version reported via the Connect API. Note: This does NOT block connections - use the User Agents tab for actual blocking.'),
             ]);
     }
 
     public function table(Table $table): Table
     {
         return $table
             ->recordTitleAttribute('title')
+            ->description('Manage emulator version information reported to clients. The minimum version setting here is informational only and does NOT block connections. To actually block old versions, use the User Agents tab.')
```

Comment:
Might be worth noting that this information is purely informational for any clients not using the DLL.